packages/tempus/src/Tempus_StepperExplicit_decl.hpp

```diff
@@ -13,6 +13,24 @@
 #include "Tempus_Stepper.hpp"
 
 
+template<class Scalar>
+class ExplicitODEParameters
+{
+  public:
+    /// Constructor
+    ExplicitODEParameters()
+      : timeStepSize_(Scalar(0.0)), stageNumber_(0)
+    {}
+    /// Constructor
+    ExplicitODEParameters(Scalar timeStepSize, int stageNumber = 0)
+      : timeStepSize_(timeStepSize), stageNumber_(stageNumber)
+    {}
+
+    Scalar                                timeStepSize_;
+    int                                   stageNumber_;
```

Comment:
Prefer setting default values like this, rather than in the constructor.
```c++
Scalar timeStepSize_ = 0.0;
int stageNumber_ = 0;
```
Then you can declare the default constructor like this:
```c++
ExplicitODEParameters() = default;
```